Makefile

```diff
@@ -15,6 +15,12 @@
 #-----------------------------------------------------------------------------
 # Global Variables
 #-----------------------------------------------------------------------------
+
+# Invoke make with DEBUG=<anything> will have the binaries built with debugger information
+ifdef DEBUG
+	export DEBUG_MODULES = ALL
```

Comment:
I'd be more inclined to keep this as a 0 vs 1 (similar to how STATIC is handled in gobuild.sh [I'd favor more unique names like GOSTATIC and GODEBUG, not to suggest that you be the one to rename STATIC]), or possibly just set/export GCFLAGS directly (I'm torn on this option  because it does make things simpler but moves config info out of the script [though I wouldn't mind seeing the script go away] ).

Reply:
I was wondering why gobuid.sh is actually needed at all. Since it's invoked in quite a few places, I'd like to look at removing it in a different issue/PR if you are ok. For the time being, I'd like to keep gcflags inside gobuild.sh

Reply:
I wonder why most of the scripts exist.  I didn't have in mind for you to be responsible to remove it.